internal/kafka/internal/services/kafka_test.go

```diff
@@ -2204,32 +2205,112 @@ func Test_KafkaService_CountByStatus(t *testing.T) {
 	}
 }
 
-func Test_KafkaService_CountByRegionAndInstanceType(t *testing.T) {
+func Test_KafkaService_CountStreamingUnitByRegionAndInstanceType(t *testing.T) {
 	type fields struct {
 		connectionFactory *db.ConnectionFactory
 	}
+	supportedInstanceTypeConfig := config.KafkaSupportedInstanceTypesConfig{
+		Configuration: config.SupportedKafkaInstanceTypesConfig{
+			SupportedKafkaInstanceTypes: []config.KafkaInstanceType{
+				{
+					Id:          "standard",
+					DisplayName: "Standard",
+					Sizes: []config.KafkaInstanceSize{
+						{
+							Id:               "x1",
+							QuotaConsumed:    1,
+							CapacityConsumed: 1,
+						},
+						{
+							Id:               "x2",
+							QuotaConsumed:    2,
+							CapacityConsumed: 2,
+						},
+					},
```

Comment:
We've got mocks for these, can we use them here? https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/internal/kafka/test/mocks/supported_instance_types/kafka_instance_size.go

Reply:
I think the `github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/internal/kafka/test/mocks/supported_instance_types` package isn't importable in the `services` package.

Reply:
should be able to import it like this: 
`mocksupportedinstancetypes "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/test/mocks/supported_instance_types"`

Reply:
I guess I am importing the wrong package :(

Reply:
it was the `/blob/main` bit that was causing issues :)

Reply:
Thanks. 

Yes, it was that.